src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -314,8 +323,17 @@ export class ESLToggleable extends ESLBaseElement {
 
   /** Actions to execute on show request */
   @bind
-  protected _onShowRequest(): void {
-    this.show();
+  protected _onShowRequest(e: CustomEvent<ESLShowRequestDetails>): void {
+    const target = e.target;
+    const detail = e.detail;
+    if (!target) return;
+    let showOptions;
+    if (detail) {
+      showOptions = Object.assign({delay: detail.delay} || {}, {...detail.params} || {});
+      const ignore = detail.ignore;
+      if (typeof ignore === 'string' && (target as Element).matches(ignore) || typeof ignore === 'function' && ignore.call(target)) return;
```

Comment:
ignore call doesn't match with signature
we should pass `this` element not an e.target to ignore predicate which is considered as contextless and expects element to be passed as first argument.

+ I'd recommend to simplify this thing, or even move it to utils, something like  
 `isMathces(el: Element, matcher: undefined | selector | (el: Element) => boolean): boolean;`